doc/samples/sid_end_device.rst

```diff
@@ -1,9 +1,9 @@
-.. _sidewalk_template:
+.. _sidewalk_end_device:
 
-Sidewalk Template
-#################
+Sidewalk End Device
+###################
 
-This sample demonstrates a template for the Sidewalk End Node application.
+This sample demonstrates a end device for the Sidewalk End Device application.
 It supports Bluetooth LE, LoRa, and FSK link modes.
 External QSPI Flash is used for firmware updates.

```

Comment:
```suggestion


```

Reply:
The sample is using qspi flash for device updates, why should we remove this line?

Reply:
In my opinion, this information is already in table in sample_list but it can stay